docker not buildable due to "Package docutils (0.21.post1) not found."

**during building docker:**

```
Notice:  To update, run: pip install --upgrade pip
Creating virtualenv mqtt-io-k3hN_2-1-py3.8 in /home/runner/.cache/pypoetry/virtualenvs
Updating dependencies
Resolving dependencies...

Package docutils (0.21.post1) not found.
Error: Process completed with exit code 1.
```

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -16,6 +16,8 @@ dataclasses = { version = "^0.8", python = ">=3.6,<3.7" }
 asyncio-mqtt = "^0.8.1"
 backoff = "^1.10.0"
 confp = "^0.4.0"
+# Fix for poetry/docutils related bug
+docutils = "0.18.1"
 
 [tool.poetry.dev-dependencies]
 mock = { version = "^4.0.3", python = ">=3.6,<3.8" }

